recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -14,11 +14,12 @@ class PahoMqttcConan(ConanFile):
     options = {"shared": [True, False],
                "fPIC": [True, False],
                "ssl": [True, False],
-               "asynchronous": [True, False]}
-    default_options = {"shared": False,
+               "samples": [True, False]}
+    # static builds didn't really work until 1.3.4
+    default_options = {"shared": True,
                        "fPIC": True,
-                       "ssl": False,
-                       "asynchronous": True}
+                       "ssl": True,
```

Comment:
When ssl changed? Was it a bug, or is just following the new release?

Reply:
I think it's better that ssl is enabled by default.  I think most people would want to use secure connections.  Is there a reason for not enabling it?

Reply:
totally agree on...
- ssl being enabled by default
- static library by default
- building both asynchronous and classic interface libraries together (even though i think most people use the asynchronous apis) as long as it doesn't cause linker issues or inflate the end executable due to the linker optimizing out all unused code